apps/app/src/test/resources/include/scans/_scan.conf

```diff
@@ -108,6 +108,10 @@
        ttl = 1s
        max-size = 1
       }
+      open-mining-rounds {
```

Comment:
I wonder if we should change the config to warn on undefined endpoints unless they're explicitly specified to have no cache. Seem seasier to not miss something that way. Definitely not for this PR.